utils/obsproc/NetCDFToIodaConverter.h

```diff
@@ -93,6 +93,52 @@ namespace gdasapp {
       location += other.location;
       oops::Log::trace() << "IodaVars::IodaVars done appending." << std::endl;
     }
+    void trim(const Eigen::Array<bool, Eigen::Dynamic, 1>& mask ) {
+      int newlocation = mask.count();
+
+      Eigen::ArrayXf longitudeMasked(newlocation);
```

Comment:
just use the constructor from above to declare a new instance of IodaVars, something like:
```c++
IodaVars iodaVarsMasked(newlocation,  this->floatMetadataName, this->intMetadataName);
```

Reply:
Done